src/workflow_components/composite_action.py

```diff
@@ -27,6 +28,42 @@ def get_or_create_composite_action(path: str) -> "CompositeAction":
     return obj
 
 
+class CompositeActionInput(GraphObject):
+    __primarykey__ = "_id"
+
+    _id = Property()
+    name = Property()
+    default = Property()
+    description = Property()
+    required = Property()
+    url = Property()
+    action = RelatedTo("CompositeAction")
+
+    def __init__(self, _id: str, name: str, url: str):
+        self._id = _id
+        self.name = name
+        self.url = url
+
+    @staticmethod
+    def from_dict(obj_dict) -> "CompositeActionInput":
+        s = CompositeActionInput(
+            _id=obj_dict["_id"], name=obj_dict["name"], url=obj_dict["url"]
+        )
+
+        if "default" in obj_dict:
+            s.default = obj_dict["default"]
+
+        if "description" in obj_dict:
+            s.description = obj_dict["description"]
+
+        if "required" in obj_dict:
```

Comment:
`s.required = obj_dict.get("required") == "true"`
No need for the if else

Reply:
Changed to: `i.required = raw_str_to_bool(obj_dict.get("required", "false"))` which keeps the default to false.